temporalcli/commands.workflow_view.go

```diff
@@ -265,6 +265,51 @@ func (*TemporalWorkflowCountCommand) run(*CommandContext, []string) error {
 	return fmt.Errorf("TODO")
 }
 
-func (*TemporalWorkflowShowCommand) run(*CommandContext, []string) error {
-	return fmt.Errorf("TODO")
+func (c *TemporalWorkflowShowCommand) run(cctx *CommandContext, args []string) error {
+	// Call describe
+	cl, err := c.Parent.ClientOptions.dialClient(cctx)
+	if err != nil {
+		return err
+	}
+	defer cl.Close()
+
+	// Print history
+	iter := &structuredHistoryIter{
+		ctx:            cctx,
+		client:         cl,
+		workflowID:     c.WorkflowId,
+		runID:          c.RunId,
+		includeDetails: true,
+		follow:         c.Follow,
+		color:          !cctx.JSONOutput && !color.NoColor,
+	}
+	if !cctx.JSONOutput {
+		cctx.Printer.Println(color.MagentaString("Progress:"))
+		// TODO: Do we need to do the "time elapsed" thing? Also needs some redoing to print
+		//   result at the end, since final event is not accessible from here.
```

Comment:
Looking for any thoughts on this

Reply:
ON `workflow execute` I think, there was this "Progress" with a counting number of seconds which I removed, and if that's what this was here, I think we can remove it here too.